src/Test/contconic.jl

```diff
@@ -45,6 +45,10 @@ function _lin1test(model::MOI.ModelLike, config::TestConfig, vecofvars::Bool)
     MOI.set(model, MOI.ObjectiveSense(), MOI.MinSense)
 
     if config.solve
+        @test MOI.get(model, MOI.TerminationStatus()) == MOI.OptimizeNotCalled
+        @test MOI.get(model, MOI.PrimalStatus()) == MOI.NoSolution
```

Comment:
I'm a bit confused now. Do solvers need to return a value for `PrimalStatus()` even if `ResultCount()` is zero?
Do solvers need to return a value for `PrimalStatus(N)` if `ResultCount() < N`?